django-backend/fecfiler/web_services/dot_fec/dot_fec_submitter.py

```diff
@@ -56,15 +58,37 @@ class EFODotFECSubmitter(DotFECSubmitter):
     """Submitter class for submitting .FEC files to EFO's webload service"""
 
     def __init__(self) -> None:
-        self.fec_soap_client = Client(f"{EFO_FILING_API}/webload/services/upload?wsdl")
+        if MOCK_EFO_FILING:
```

Comment:
This still has the issue of not basing whether it's mock or not off of the mock flag in tasks.py